webpack.rules.js

```diff
@@ -1,149 +0,0 @@
-/*
- * Copyright (c) 2022,  Quansight
- *
- * This file is part of the tree-finder library, distributed under the terms of
- * the BSD 3 Clause license. The full license can be found in the LICENSE file.
- */
-const CssMinimizerPlugin = require('css-minimizer-webpack-plugin');
-const MiniCssExtractPlugin = require("mini-css-extract-plugin");
-const path = require("path");
-const TerserPlugin = require("terser-webpack-plugin");
-
-const cssLoader = () => {
-	return {
-		loader: "css-loader",
-		options: {
-			importLoaders: 1,
-			import: true,
-			modules: {
-				localIdentName: "[local]",
-			},
-		},
-	};
-}
-
-// load bitmap image rules
-const bitmapRules = [
-	{
-		test: /\.(jpg|png|gif)$/,
-		use: "file-loader"
-	},
-];
-
-// load dependency source maps
-const dependencySrcMapRules = [
-	{
-		test: /\.js$/,
-		use: "source-map-loader",
-		enforce: "pre",
-		exclude: /node_modules/,
-	},
-	{ test: /\.js.map$/, use: "file-loader" },
```

Comment:
Seems like this one was deleted. Was it not needed?

Reply:
This has now been deprecated so moved to `module assets` https://v4.webpack.js.org/loaders/file-loader/

Reply:
I'm not solid on any of this, but I thought the rules still had to stay in place, but the `use: "file-loader"` could be replaced with `type: "asset/resource"` or `type: "asset"`, which is how you moved over the SVG rules.

I'm just pattern-matching here. I see that the SVG rules have been carried over, but not the bitmap rules, nor this `.js.map` rule.

Reply:
Although I'm not sure about why `.js.map` is needed because the source maps seemed to work in all of the different ways that I built the app after applying the code changes from this PR (not just webpack dev server mode).